web-components/navdots/navdots.svelte

```diff
@@ -3,8 +3,9 @@
 <script lang="ts">
   import { createEventDispatcher } from 'svelte'
 
+  export let animateSlide: boolean = true
```

Comment:
Added an option to disable the slide animation. On the NTP we control the current dot explicitly via the `activeDot` property.